seismicpro/src/refractor_velocity/refractor_velocity.py

```diff
@@ -290,6 +290,34 @@ def from_first_breaks(cls, offsets, times, init=None, bounds=None, n_refractors=
         self.times = times
         return self
 
+    @classmethod
+    def from_file(cls, path, encoding="UTF-8"):
+        """Create a RefractorVelocity instance from a file.
+
+        File should have coords and parameters of a single RefractorVelocity with next structure:
+         - The first row contain the Coordinates parameters names (name_x, name_y, coord_x, coord_y) and
+        the RefractorVelocity parameters names ("t0", "x1"..."x{n-1}", "v1"..."v{n}", "max_offset").
+         - The second row contains the coords names, coords values, and parameters values of a RefractorVelocity.
```

Comment:
If you'll change this part, please unify other docs as well.

```suggestion
        File should have coords and parameters of a single RefractorVelocity with the following structure:
         - The first row contains names of Coordinates parameters (name_x, name_y, coord_x, coord_y) and
        the RefractorVelocity parameters ("t0", "x1"..."x{n-1}", "v1"..."v{n}", "max_offset").
         - The second row contains the coords names, coords values, and parameters values of a RefractorVelocity.
```